### PR TITLE
chore(wallet-ui): initialize project

### PR DESCRIPTION
**Pull request by gregfromstl**
## PR-Codex overview
This PR enhances the Thirdweb Wallet UI by configuring Next.js, Tailwind CSS, and BiomeJS, updating dependencies, setting up TypeScript, and improving global styles.

### Detailed summary
- Configured Next.js for Wallet UI
- Added Tailwind CSS configuration
- Updated dependencies for Next.js, React, and TypeScript
- Implemented global styles using CSS variables
- Set up TypeScript for type safety

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by gregfromstl** [comment]
* **#3477** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/3477?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gregfromstl and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/wallet-ui/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jnsdls) [inline apps/wallet-ui/README.md:1]: I was trying to `README` but it was empty

File: apps/wallet-ui/biome.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.1/schema.json",
+  "extends": ["../../biome.json"],
+  "linter": {
+    "rules": {
+      "style": {}
+    }
+  },
```
Comment (jnsdls) [inline apps/wallet-ui/biome.json:8]: I _believe_ this is the same as `linter:true`?

File: apps/wallet-ui/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "wallet-ui",
```
Comment (jnsdls) [inline apps/wallet-ui/package.json:2]: make sure to also add this to the `ignore` in the changesets config file

File: apps/wallet-ui/src/app/[ecosystem]/layout.tsx [hunk 4]
```diff
@@ -0,0 +1 @@
+export default function Layout() {}
```
Comment (jnsdls) [inline apps/wallet-ui/src/app/[ecosystem]/layout.tsx:1]: shouldn't add empty files

File: pnpm-lock.yaml [hunk 5]
```diff
@@ -38719,6 +38759,33 @@ snapshots:
     transitivePeerDependencies:
       - ts-node
 
+  tailwindcss@3.4.4(ts-node@10.9.2(@types/node@20.14.6)(typescript@5.5.2)):
```
Comment (jnsdls) [inline pnpm-lock.yaml:38762]: suspicious, do we have different versions of tailwind installed now - if so lets align them

**Review by jnsdls — CHANGES_REQUESTED** [review]
couple of things and then also update the favicon (we're not vercel)

**Review by jnsdls — APPROVED** [review]

**Pull request merged**